### DISCUSSION TITLE
BT ID's on notices

### DISCUSSION THREAD

**Discussion by lillabeatak** [OP-TED/eForms-SDK#616] [Ideas]
The notices published on TED do not contain the BT ID's. What is the reason for? It would be recommended that the notices on TED contain BT ID's.

**Comment by bertrand-lorentz** [comment]
As this doesn't seem to be an issue with the SDK itself, I've converted this to a discussion topic.

I'm sure I understand what you mean by "notices on TED contain BT ID's". Could you expand on this ?

**Reply by lillabeatak** [reply]
I meant, that on the eNotices2 interface, users can also see the field's BT identifiers, at the naming of each fields. In case of notices already published on TED, only the name of the field is visible, not contain the field's BT identifier. For easier identification, it would be worthwhile to put up the BT identifier.
